### ISSUE TITLE
Support non-https urls for lazy people

### ISSUE THREAD

**Issue by vladikoff** [potch/foxshot#1]
```
✗ foxshot theverge.com  
No valid dimensions provided, defaulting to 1024x768
loading site...
Error during capture: { InvalidArgumentError: Malformed URL: theverge.com is not a valid URL.
    at Object.throwDecodedError (/Users/vladikoff/.nvm/versions/node/v8.9.0/lib/node_modules/foxshot/node_modules/selenium-webdriver/lib/error.js:514:15)
    at parseHttpResponse (/Users/vladikoff/.nvm/versions/node/v8.9.0/lib/node_modules/foxshot/node_modules/selenium-webdriver/lib/http.js:519:13)
    at doSend.then.response (/Users/vladikoff/.nvm/versions/node/v8.9.0/lib/node_modules/foxshot/node_modules/selenium-webdriver/lib/http.js:441:30)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
From: Task: WebDriver.navigate().to(theverge.com)
    at thenableWebDriverProxy.schedule (/Users/vladikoff/.nvm/versions/node/v8.9.0/lib/node_modules/foxshot/node_modules/selenium-webdriver/lib/webdriver.js:807:17)
    at Navigation.to (/Users/vladikoff/.nvm/versions/node/v8.9.0/lib/node_modules/foxshot/node_modules/selenium-webdriver/lib/webdriver.js:1133:25)
    at thenableWebDriverProxy.get (/Users/vladikoff/.nvm/versions/node/v8.9.0/lib/node_modules/foxshot/node_modules/selenium-webdriver/lib/webdriver.js:988:28)
    at runTest (/Users/vladikoff/.nvm/versions/node/v8.9.0/lib/node_modules/foxshot/capture.js:62:16)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
  name: 'InvalidArgumentError',
  remoteStacktrace: 'stack backtrace:\n   0:        0x107887a8e - backtrace::backtrace::trace::h69682bcb53decaf6\n   1:        0x107887acc - backtrace::capture::Backtrace::new::hdc9d731a957304a6\n   2:        0x1077b8434 - webdriver::error::WebDriverError::new::h930061353eac4c81\n   3:        0x1077c201e - geckodriver::marionette::MarionetteSession::response::h451801d534e80d5b\n   4:        0x1077a109b - _$LT$webdriver..server..Dispatcher$LT$T$C$$u20$U$GT$$GT$::run::hfda25a6dc0b512aa\n   5:        0x107777645 - std::sys_common::backtrace::__rust_begin_short_backtrace::h95009c1d3a320838\n   6:        0x10777f11d - std::panicking::try::do_call::h8c5f07f1fc714fb2\n   7:        0x1079389cc - __rust_maybe_catch_panic\n   8:        0x107794f55 - _$LT$F$u20$as$u20$alloc..boxed..FnBox$LT$A$GT$$GT$::call_box::h6a96e09ff4d37bff\n   9:        0x107934c3b - std::sys::imp::thread::Thread::new::thread_start::h823686b907c11c46\n  10:     0x7fff7808e6c0 - _pthread_body\n  11:     0x7fff7808e56c - _pthread_start' }
```

**Comment by potch** [comment]
asking me to parse URLs eh...

**Comment by potch** [comment]
fixed by 9c3c09357103956bc9310c3907fad08c6b2e9c3d

**Comment by vladikoff** [comment]
Hero!
